src/Data.php

```diff
@@ -274,4 +248,20 @@ protected function keyToPathArray(string $path): array
 
         return \explode('.', $path);
     }
+
+    /**
+     * @param string|string[] $path
+     *
+     * @return string
+     *
+     * @psalm-pure
+     */
+    protected static function formatPath($path): string
+    {
+        if (is_string($path)) {
+            $path = self::keyToPathArray($path);
+        }
+
+        return implode(' » ', $path);
```

Comment:
These formatted paths will be used in exception messages which may be shown to an end-user. For example, once #29 is implemented, code like this:

```php
$data = new Data();
$data->get('sculpin.permalink'); // does not exist, and no default provided
```

Will result in a `MissingPathException` with a message like this:

```
No data exists at the given path: "sculpin » permalink"
```

I think representing the path as `sculpin » permalink` instead of `sculpin.permalink` or `scuplin/permalink` makes it a little clearer to end-users that the error has to do with some kind of nested data (especially if they aren't aware that this library is being used under-the-hood.